src/org/labkey/test/components/react/MultiMenu.java

```diff
@@ -27,6 +27,22 @@ protected MultiMenu(WebElement element, WebDriver driver)
         super(driver, element);
     }
 
+    private WebElement getMenuList()
+    {
+        return Locator.tagWithClass("ul", "dropdown-menu").findElement(this);
+    }
+
+    public WebElement getMenuItem(String text)
+    {
+        return Locator.tag("li").withChild(Locator.tagWithAttribute("a", "role", "menuitem")
+                .withText(text)).findElement(getMenuList());
+    }
+
+    public boolean isMenuItemDisabled(String text)
+    {
+        return getMenuItem(text).getAttribute("class").toLowerCase().contains("disabled");
```

Comment:
You might be able to call:
`return !getMenuItem(text).isEnabled();`

Reply:
Unfortunately that didn't work when I tried it.